src/multi.rs

```diff
@@ -203,6 +208,32 @@ impl MultiState {
         extra_lines: Option<Vec<String>>,
         now: Instant,
     ) -> io::Result<()> {
+        // Reap all consecutive 'zombie' progress bars from head of elements
+        let mut zombies = vec![];
+        let mut adjust = 0;
+        for index in self.ordering.iter() {
+            let element = &self.elements[*index];
+            if element.is_zombie {
```

Comment:
Can we change this to `if !element.is_zombie { break; }` to reduce rightward drift?